### ISSUE TITLE
Unit Detail improvements

### ISSUE THREAD

**Issue by martinacantaro** [lambdaclass/champions_of_mirra#159]
- Show level of the unit
- Show stats: shield, speed, attack, HP (depends on #102)
- Show quality
- Remove the two lower item boxes which don't have a functionality in the Alpha
- Lore button: it opens a popup (or another scene) with the lore of the character and an image of the character

Lower priority:
- Make the left/right buttons work to browse through the champions
